sourced/ml/mining/models/dependencies.py

```diff
@@ -0,0 +1,99 @@
+from modelforge import (
+    assemble_sparse_matrix,
+    disassemble_sparse_matrix,
+    merge_strings,
+    Model,
+    register_model,
+    split_strings,
+)
+from sourced.ml.core.models.license import DEFAULT_LICENSE
+
+
+@register_model
+class Dependencies(Model):
+    """
+    Occurence matrix between files and dependencies.
+    """
+
+    NAME = "deps"
+    VENDOR = "source{d}"
+    DESCRIPTION = (
+        "This model contains the (binary) cooccurrence matrix between dependencies and files,"
```

Comment:
```suggestion
        "Binary cooccurrence matrix between dependencies and files,"
```
(sorry; this is after the second read)

Reply:
no problem